shell/browser/api/electron_api_web_contents.cc

```diff
@@ -1716,10 +1716,8 @@ void WebContents::DidStopLoading() {
   Emit("did-stop-loading");
 }
 
-bool WebContents::EmitNavigationEvent(
-    const std::string& event,
+content::RenderFrameHost* WebContents::GetRenderFrameHost(
```

Comment:
This can be moved out of the WebContents class and into the anonymous `namespace { ... }` near the top of the file.